include/E57Simple.h

```diff
@@ -0,0 +1,881 @@
+                ( principalPointY == rhs.principalPointY );
+      }
+      bool operator!=( const CylindricalRepresentation &rhs ) const
+      {
+         return !operator==( rhs );
+      }
+   };
+
+   //! @brief Stores an image from a camera
+   struct E57_DLL Image2D
+   {
+      ustring name;        //!< A user-defined name for the Image2D.
+      ustring guid;        //!< A globally unique identification string for the current version of the Image2D object
+      ustring description; //!< A user-defined description of the Image2D
+      DateTime acquisitionDateTime; //!< The date and time that the image was taken
+
+      ustring associatedData3DGuid; //!< The globally unique identification string (guid element) for the Data3D that
+                                    //!< was being acquired when the picture was taken
+
+      ustring sensorVendor; //!< The name of the manufacturer for the sensor used to collect the points in this Data3D.
+      ustring sensorModel;  //!< The model name or number for the sensor.
+      ustring sensorSerialNumber; //!< The serial number for the sensor.
+
+      RigidBodyTransform pose; //!< A rigid body transform that describes the coordinate frame of the camera in the
+                               //!< file-level coordinate system
+
+      VisualReferenceRepresentation
+         visualReferenceRepresentation; //!< Representation for an image that does not define any camera projection
+                                        //!< model. The image is to be used for visual reference only
+      PinholeRepresentation
+         pinholeRepresentation; //!< Representation for an image using the pinhole camera projection model
+      SphericalRepresentation
+         sphericalRepresentation; //!< Representation for an image using the spherical camera projection model.
+      CylindricalRepresentation
+         cylindricalRepresentation; //!< Representation for an image using the cylindrical camera projection model
+   };
+
+   //! @brief Identifies the format representation for the image data
+   enum Image2DType
+   {
+      E57_NO_IMAGE = 0,      //!< No image data
+      E57_JPEG_IMAGE = 1,    //!< JPEG format image data.
+      E57_PNG_IMAGE = 2,     //!< PNG format image data.
+      E57_PNG_IMAGE_MASK = 3 //!< PNG format image mask.
+   };
+
+   //! @brief Identifies the representation for the image data
+   enum Image2DProjection
+   {
+      E57_NO_PROJECTION = 0, //!< No representation for the image data is present
+      E57_VISUAL = 1,        //!< VisualReferenceRepresentation for the image data
+      E57_PINHOLE = 2,       //!< PinholeRepresentation for the image data
+      E57_SPHERICAL = 3,     //!< SphericalRepresentation for the image data
+      E57_CYLINDRICAL = 4    //!< CylindricalRepresentation for the image data
+   };
+
+   ////////////////////////////////////////////////////////////////////
+
+   //! @brief Used for reading of the E57 file with E57 Simple API
+   class E57_DLL Reader
```

Comment:
Would it make sense to put reader in its own header (`E57SimpleReader.h`), writer in its own header (`E57SimpleWriter.h`) and change `E57Simple.h` to `E57SimpleData.h` (or something like that?)